src/main/scala/com/microsoft/hyperspace/util/LogicalPlanUtils.scala

```diff
@@ -35,4 +42,35 @@ object LogicalPlanUtils {
       case _ => false
     }
   }
+
+  def getAllAvailableIndexes(states: Seq[String] = Seq()): Seq[IndexLogEntry] = {
+    Hyperspace
+      .getContext(SparkSession.getActiveSession.get)
+      .indexCollectionManager
+      .getIndexes(states)
+  }
+
+  def getCandidateIndexesForPlan(plan: LogicalPlan): Seq[IndexLogEntry] = {
+    val signatureMap: mutable.Map[String, String] = mutable.Map()
```

Comment:
Let's retain the existing comment (reworded a bit) and remove type for a local variable:
```scala
// Map of a signature provider to a signature generated for the given plan.
val signatureMap = mutable.Map[String, String]()
```